easybuild/framework/extensioneasyblock.py

```diff
@@ -105,11 +116,11 @@ def run(self, unpack_src=False):
             targetdir = os.path.join(self.master.builddir, remove_unwanted_chars(self.name))
             self.ext_dir = extract_file(self.src, targetdir, extra_options=self.unpack_options,
                                         change_into_dir=False)
-            change_dir(self.ext_dir)
 
-            if self.start_dir and os.path.isdir(self.start_dir):
-                self.log.debug("Using start_dir: %s", self.start_dir)
-                change_dir(self.start_dir)
+            self._set_start_dir()
+            change_dir(self.start_dir)
```

Comment:
Shouldn't this be
```
change_dir(self.cfg['start_dir'])
```
since you never alter the value of `self.start_dir`?

Reply:
`self.start_dir` is a property which accesses `self.cfg['start_dir']` so they are synonym (but you can't set the former)

Reply:
Indeed, see https://github.com/easybuilders/easybuild-framework/blob/186eaf9a243cdea4a0c147cada6e3c6b8239db2e/easybuild/framework/easyblock.py#L1569-L1572 (this is inherited here since `ExtensionEasyBlock` derives from `EasyBlock`